### PR TITLE
Updating project infrastructure

### PR DESCRIPTION
**Pull request by lucianopf**
# Description 

This PR aims to update the project to the new infrastructure and also updates how we handle CI/CD changing from TravisCI to CircleCI.

Relates to https://github.com/pagarme/ghostbusters/issues/572

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fb89aec8ba8550e1b741030da7ede3b12e0d0a37
  * Commits must be in the following format: **%{scope}: %{description}**

Check out our Git Style Guide here: https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 65ef6fe12cbd1676f51b33a654c2398aa9f15e8b
  * Commits must be in the following format: **%{scope}: %{description}**

Check out our Git Style Guide here: https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: src/lib/instrumentation/index.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-  if (process.env.NODE_ENV === 'production') {
-    ddTrace.init()
+  if (process.env.NODE_ENV === 'production' && process.env.STAGE !== 'stg') {
+    ddTrace.init({
+      logInjection: true,
+      runtimeMetrics: true,
+    })
 
-    if (process.env.NEWRELIC_KEY) {
+    if (process.env.NEWRELIC_KEY && process.env.NEWRELIC_KEY !== 'REDACTED') {
```
Comment (mccraveiro) [inline src/lib/instrumentation/index.js:10]: :eyes:

Comment (lucianopf) [reply src/lib/instrumentation/index.js:10]: Hahaha, isso foi um semi-teste, mas de qualquer forma não implica em nada crítico ^^

Tem opiniões sobre isso mestre @mccraveiro ? 😬

**Pull request merged**